src/test/regress/expected/view_propagation.out

```diff
@@ -359,13 +359,93 @@ ERROR:  cannot change name of view column "id" to "a"
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+CREATE VIEW alter_view_1 AS SELECT * FROM alter_view_table;
+-- Set/drop default value is not supported by Citus
+ALTER VIEW alter_view_1 ALTER COLUMN val1 SET DEFAULT random()::text;
+ERROR:  Citus doesn't support setting or resetting default values for a column of view
+ALTER TABLE alter_view_1 ALTER COLUMN val1 SET DEFAULT random()::text;
+ERROR:  Citus doesn't support setting or resetting default values for a column of view
+ALTER VIEW alter_view_1 ALTER COLUMN val1 DROP DEFAULT;
+ERROR:  Citus doesn't support setting or resetting default values for a column of view
+ALTER TABLE alter_view_1 ALTER COLUMN val1 DROP DEFAULT;
+ERROR:  Citus doesn't support setting or resetting default values for a column of view
+-- Set/reset options view alter view/alter table commands
+ALTER VIEW alter_view_1 SET (check_option=cascaded);
+ALTER VIEW alter_view_1 SET (security_barrier);
+ALTER VIEW alter_view_1 SET (check_option=cascaded, security_barrier);
+ALTER VIEW alter_view_1 SET (check_option=cascaded, security_barrier = true);
+ALTER TABLE alter_view_1 SET (check_option=cascaded);
+ALTER TABLE alter_view_1 SET (security_barrier);
+ALTER TABLE alter_view_1 SET (check_option=cascaded, security_barrier);
+ALTER TABLE alter_view_1 SET (check_option=cascaded, security_barrier = true);
+\d+ alter_view_1
+                    View "view_prop_schema.alter_view_1"
+ Column |  Type   | Collation | Nullable | Default | Storage  | Description
+---------------------------------------------------------------------
+ id     | integer |           |          |         | plain    |
+ val1   | text    |           |          |         | extended |
+View definition:
+ SELECT alter_view_table.id,
+    alter_view_table.val1
+   FROM alter_view_table;
+Options: check_option=cascaded, security_barrier=true
+
+\c - - - :worker_1_port
+SET search_path to view_prop_schema;
+\d+ alter_view_1
+                    View "view_prop_schema.alter_view_1"
+ Column |  Type   | Collation | Nullable | Default | Storage  | Description
+---------------------------------------------------------------------
+ id     | integer |           |          |         | plain    |
+ val1   | text    |           |          |         | extended |
+View definition:
+ SELECT alter_view_table.id,
+    alter_view_table.val1
+   FROM alter_view_table;
+Options: check_option=cascaded, security_barrier=true
+
+\c - - - :master_port
+SET search_path to view_prop_schema;
+ALTER TABLE alter_view_1 RESET (check_option, security_barrier);
+ALTER VIEW alter_view_1 RESET (check_option, security_barrier);
+-- Change the schema of the view
+ALTER TABLE alter_view_1 SET SCHEMA view_prop_schema_inner;
+ALTER VIEW view_prop_schema_inner.alter_view_1 SET SCHEMA view_prop_schema;
+-- Rename view and view's column name
+ALTER VIEW alter_view_1 RENAME COLUMN val1 TO val2;
```

Comment:
would be good to show that after `rename column` we can still de-parse the alias of the view properly with the new name. For example, we can re-sync the metadata (or only get the metadata events) and show  that alias is correct